packages/server/src/queue/queue.service.ts

```diff
@@ -34,26 +41,60 @@ export class QueueService {
     if (queueSize === 0) {
       throw new NotFoundException();
     }
-    return QuestionModel.openInQueue(queueId)
-      .leftJoinAndSelect('question.creator', 'creator')
-      .leftJoinAndSelect('question.taHelped', 'taHelped')
-      .getMany();
+
+    const questionsFromDb = await QuestionModel.find({
+      relations: ['creator', 'taHelped'],
+      where: {
+        queueId,
+        status: In(StatusSentToCreator),
```

Comment:
shouldn't it be In( ) of all the statuses in queue, help, and priorityQueue?

Reply:
`StatusSentToCreator` includes all of those

Reply:
ah yes but we dont' need them in th is function fuck

Reply:
oh yeah but we don't need them in this funciton fuck